app/_data/docs_nav_konnect.yml

```diff
@@ -49,49 +49,37 @@
   items:
     - text: Overview
       url: /runtime-manager
-    - text: Runtime Groups
-      items:
-        - text: Overview
-          url: /runtime-manager/runtime-groups
-        - text: Dashboard
-          url: /runtime-manager/runtime-groups/dashboard
-        - text: Manage Runtime Groups with UI
-          url: /runtime-manager/runtime-groups/manage
-        - text: Manage Runtime Groups with decK
-          url: /runtime-manager/runtime-groups/declarative-config
     - text: Runtime Instances
       items:
-        - text: Installation Options
+        - text: Overview
           url: /runtime-manager/runtime-instances/
-        - text: Install with Docker
-          url: /runtime-manager/runtime-instances/gateway-runtime-docker
-        - text: Install on Kubernetes
-          url: /runtime-manager/runtime-instances/gateway-runtime-kubernetes
-        - text: Install on Linux
-          url: /runtime-manager/runtime-instances/gateway-runtime-conf
-        - text: Install on AWS
-          url: /runtime-manager/runtime-instances/gateway-runtime-aws
-        - text: Install on Azure
-          url: /runtime-manager/runtime-instances/gateway-runtime-azure
-        - text: Upgrade a Runtime Instance to a New Version
+        - text: Upgrade a Runtime Instance
           url: /runtime-manager/runtime-instances/upgrade
-        - text: Renew Certificates
+        - text: Verify a Runtime Instance
+          url: /runtime-manager/runtime-instances/verify-instance
+        - text: Renew Data Plane Certificates
           url: /runtime-manager/runtime-instances/renew-certificates
         - text: Runtime Parameter Reference
           url: /runtime-manager/runtime-instances/runtime-parameter-reference
-    - text: Create Consumer Groups
-      url: /runtime-manager/consumer-groups
-    - text: Secrets Management
+    - text: Runtime Configuration
       items:
         - text: Overview
-          url: /runtime-manager/vaults
-        - text: Set Up and Use a Vault in Konnect
-          url: /runtime-manager/vaults/how-to
-    - text: Kong Gateway Configuration in Konnect
-      url: /runtime-manager/gateway-config
-    - text: Plugin Ordering Reference
-      url: /runtime-manager/plugins
-    - text: Troubleshoot
+          url: /runtime-manager/configuration
+        - text: Create Consumer Groups
+          url: /runtime-manager/configuration/consumer-groups
+        - text: Secrets Management
+          items:
+            - text: Overview
+              url: /runtime-manager/configuration/vaults
+            - text: Set Up and Use a Vault in Konnect
+              url: /runtime-manager/configuration/vaults/how-to
+    - text: Manage Runtime Groups using decK
```

Comment:
Would this page fit better in the "Runtime Configuration" section?

Reply:
You know? I think you're right. Because what decK is doing is exactly that - managing config.